src/org/rascalmpl/repl/rascal/RascalInterpreterREPL.java

```diff
@@ -135,10 +136,14 @@ public IDEServices initialize(Reader input, PrintWriter stdout, PrintWriter stde
         return services;
     }
 
-
     @Override
     public ICommandOutput handleInput(String command) throws InterruptedException, ParseError, StopREPLException {
         Objects.requireNonNull(eval, "Not initialized yet");
+        if (command.equals("\n")) {
```

Comment:
this should be done at a higher level. as we want the same functionality for all rascal REPLs. so that later the compiled repl also has this functionality.

The `RascalInterpreterREPL` class should only contain code that needs to reference the interpreter classes/state.

Reply:
I have now lifted this logic to the `BaseREPL` level.

Reply:
I should have been more clear, I meant 1 higher level, not the highest level. as this behavor should align with the input parser specified.